.vscode/settings.json

```diff
@@ -23,7 +23,6 @@
     "common/temp": true
   },
   "files.trimTrailingWhitespace": true,
-  "typescript.tsdk": "node_modules\\typescript\\lib",
```

Comment:
why is this no longer needed?

Reply:
It's not so much that it's just now no longer needed, it's just that this has been broken for a very long time (at least since the move to rush), and I figured I might as well remove it.

I considered actually fixing this (pointing to the typescript install in common/temp/node_modules), but I figured that using VS Code's built-in TS version has been working well enough so far, and is probably a bit more stable...